### ISSUE TITLE
🐛 Bug: curly is changed even in prettier-ignore block

### ISSUE THREAD

**Issue by HolgerJeromin** [JoshuaKGoldberg/prettier-plugin-curly#625]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/prettier-plugin-curly/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

This should be kept verbatim:
```ts
// prettier-ignore
function ignored() {
    if(foo  .bar)
        return
}
```

### Actual

The code is changed to:
```ts
// prettier-ignore
function ignored() {
    if(foo.bar){
return}
}
```

### Additional Info

Perhaps this is a bug in prettier that no plugins should be called in ignored code...
I am happy to report it over there if you are not able to transfer.

**Comment by JoshuaKGoldberg** [comment]
Yeah I'm not sure whose responsibility it is to handle these comments. My hunch is that at the very least this is a documentation issue on Prettier core. There's no recommendation I could find -e.g. in https://prettier.io/docs/en/plugins.html- on the "right" way to do it. Yes please on the reporting it over there. 🙂

**Comment by HolgerJeromin** [comment]
Reported as https://github.com/prettier/prettier/issues/16951

**Comment by JoshuaKGoldberg** [comment]
Thanks to https://github.com/prettier/prettier/issues/16951#issuecomment-2557976588, it looks like we have a preferred solution: switching this plugin to overwrite the estree plugin's `printer.print`. Accepting PRs!
